clients/js/src/index.ts

```diff
@@ -126,9 +125,10 @@ export class Collection {
         );
       }
     }
+    if (embeddings === undefined) throw new Error("embeddings is undefined but shouldnt be")
```

Comment:
Does this only occur if the embedding function errors?

Reply:
Yes, you are correct about the logic. Is that worth having or is this "over-validating"?